### ISSUE TITLE
Database will not open after multiple attempts of fixing it

### ISSUE THREAD

**Issue by AmtrakBF** [apache/couchdb#588]
## Expected Behavior

The service is running and you can connect to your database
## Current Behavior

I watched a few videos on how to set it up and it followed them step by step, I go to connect to the database and it gives me:
 This site can’t be reached

127.0.0.1 refused to connect.
Try:
Checking the connection
Checking the proxy and the firewall
ERR_CONNECTION_REFUSED

I am using windows 10 creators update
## Possible Solution

I tried to Shift+Right Click on the bin folder and run command prompt, I then typed couchdb and it ran its thing... Didn't work so I tried to run services as an admin and start the service "Apache Couchdb"  and to my luck, it didn't start. I tried to replace the 'nssm.exe' file and I haven't gotten it to work yet. 
## Steps to Reproduce (for bugs)

1. When I go to start the program it gives me an error "Windows could not start the Apache CouchDB on Local Computer. For more information, review the System Event Log. If this is a non-Microsoft service, contact the service vendor, and refer to service-specific error code 3.
2. When I connect to 127.0.0.1:5984/_utils I get a "refused to connect" 

## Context

I can't use the database
## Your Environment

* Version used: CouchDB 2.0.0.1
* Browser Name and version: Chrome 58.0.3029.110 (64-bit)
* Operating System and version (desktop or mobile): Windows 10 Creators Update
* Link to your project: NULL

**Comment by wohali** [comment]
This is a duplicate of https://github.com/apache/couchdb/issues/508 . We hope to fix this problem with the CouchDB 2.1 release.

**Comment by MtDalPizzol** [comment]
I'm facing this with CouchDB 2.1.0 on Linux Mint 18.1 (Ubuntu 16.04 Xenial). The service seems to be running but the REST API and Fauxton are unreachable. I described the problem with more details here, but got no ideas. What is strange is that, in my case, everything was working just fine. It simply "exploded" after I inserted a new doc in the _users database and synced it with a PouchDB storage.
